dataservice/api/common/model.py

```diff
@@ -14,6 +14,11 @@
 COMMON_ENUM = {"Not Reported", "Not Applicable", "Not Allowed To Collect",
                "Not Available", "Reported Unknown"}
 
+VISIBILITY_REASON_ENUM = {
+    "Ready For Release", "Pre-Release", "Peddy Issue", "Consent Hold",
+    "File Quality Issue", "Other", "Unknown"
+}
```

Comment:
Suggested additional reasons:
1. "NGSCheckmate mismatch" (similar to "Peddy Issue", but for tumors). I might suggest using "Peddy mismatch" or "Familial mismatch" as we might run additional algorithms to support Peddy, instead)
2. "Misidentified sample or sample swap" - hard to delineate the two, so we can batch this.

Reply:
>Is "NGSCheckmate mismatch" something that can we can categorize under Peddy Issue or is it not actually a pedigree related issue? If it is not a pedigree related issue, are you suggesting that it have its own enum value for visibility_reason?

NGSCheckmate is another genomic matching tool but it is not a pedigree tool like Peddy - it is used for tumor/normal and tumor/tumor matching of samples prior to running paired normal/tumor analyses or RNA-Seq analyses, so I would categorize it differently, or broaden the `visibility_reason == Peddy issue` to be something like `"Sample Issue"` as you noted above and then `visibility_comment` can be any of: "Peddy mismatch", "PLINK mismatch" (we will run this on PBTA for sure in the future), or "NGSCheckmate mismatch".

>Can you define what "Misidentified Sample" and "Sample Swap" would be? Could we create a new reason called something like Sample Issue and the visibility_comment for each would then be "Misidentified Sample" / "Sample Swap"

These are cases that while a tumor/normal match seems to have occurred, the actual diagnosis does not match the pathology report or path-banked specimens - I will go over these with our pathologists if I find that the data tells a [drastically] different story than the reported diagnosis. For instance, we have a case which we hid `BS_JXF8A2A6` in 12/2020 because it is listed as a craniopharyngioma, but during analysis, we discovered it has a hallmark RNA event which would suggest it was a Ewing Sarcoma. This is highly unlikely. Pathologist reran a stain on a slide confirming the sample she has is in fact cranio, so ours must be a sample swap either at the seq center or in the BioRC. Cut to yesterday ... I discovered the methylation id for this sample also subtyped as a ewing sarcoma `BS_SRW0DH5G`, so I believe now that this is a bioRC issue in that our banked sample is not what it should be. Hence, mis-identified or sample swap. So, you can lump all of this into "Sample Issue", sure - just wanted to be sure the descriptors are available somewhere!

Reply:
@jharenza Thank you so much! I'll add `Sample Issue` as its own reason and then as you suggested we can use the `visibility_comment` to capture any of the following: "PLINK mismatch", "NGSCheckmate mismatch", etc